### DISCUSSION TITLE
Missing Temperature Info

### DISCUSSION THREAD

**Discussion by oosthub** [kellerza/sunsynk#221] [General]
Hi,

I used to use the Solarman integration which had temperate info for AC DC and Battery.... I cannot find those... Are they missing or am i Blind? If they are missing, are there any plan to add them?

 sensor.solarman_battery_temperature
sensor.solarman_ac_temperature
sensor.solarman_dc_temperature

Loving the integration btw.

**Comment by oosthub** [comment]
Found it
Missed it in the documentation

**Reply by kellerza** [reply]
Glad you found it. I’m not sure you get usable values on all inverters
